client/state/purchases/selectors/get-included-domain-transfer.js

```diff
@@ -0,0 +1,29 @@
+import { isDomainTransfer } from '@automattic/calypso-products';
+import { find } from 'lodash';
+import { isSubscription } from 'calypso/lib/purchases';
+import { getSitePurchases } from './get-site-purchases';
+
+import 'calypso/state/purchases/init';
+
+/**
+ * Returns a purchase object that corresponds to that subscription's transferred domain
+ *
+ * Even if a domain transfer was purchased with the subscription, it will
+ * not be returned if the domain transfer product was paid for separately (eg: if it was
+ * renewed on its own).
+ * @param   {Object} state  global state
+ * @param   {Object} subscriptionPurchase  subscription purchase object
+ * @returns {Object} domain transfer purchase if there is one, null if none found or not a subscription object passed
+ */
+export const getIncludedDomainTransfer = ( state, subscriptionPurchase ) => {
+	if ( ! subscriptionPurchase || ! isSubscription( subscriptionPurchase ) ) {
+		return null;
+	}
+
+	const { includedDomain } = subscriptionPurchase;
+	const sitePurchases = getSitePurchases( state, subscriptionPurchase.siteId );
+	return find(
+		sitePurchases,
+		( purchase ) => isDomainTransfer( purchase ) && includedDomain === purchase.meta
+	);
```

Comment:
There's a concerted effort to remove lodash from our codebase as it creates a lot of bloat and for the most part it's unnecessary. In this case, JavaScript's own `Array.prototype.find` should be enough for what we need.

```suggestion
	return sitePurchases.find(
		( purchase ) => isDomainTransfer( purchase ) && includedDomain === purchase.meta
	);
```

Reply:
Again, me doing brainless copy/paste. I'll adapt. Nice catch!

Reply:
Done in 17e7b9f1c7ead655b0e3ee8d424364841cd4d78d